siac/hostcmd.go

```diff
@@ -51,6 +52,61 @@ Doing so will override the standard connectivity checks.`,
 	}
 )
 
+// parseSize converts strings of form 10GB to a size in bytes
+// If parsing fails the original value is returned unmodified
+func parseSize(value string) string {
+	var suffix string
+	var size int
```

Comment:
typically you declare function variables just before you use them. I'd probably put these directly above the first `if` statement.